### PR TITLE
feat(forms): add ng-pending CSS class during async validation

### LINKED ISSUE #10336: Adding a ng-pending class during the pending phase of an input with async validation
**I'm submitting a**
feature request

**Current behavior**
There is no ng-pending class during the pending state of a input with async validation

**Expected/desired behavior**
Add a ng-pending class during the pending state of a input with async validation

**Reproduction of the problem**
As soon a you type in the field you get an ng-invalid, but the state is not invalid it should be pending.
https://plnkr.co/edit/ZwAKRn?p=preview

**What is the expected behavior?**

**What is the motivation / use case for changing the behavior?**
If would be handy to differentiate the phases on the inputs via css  

**Please tell us about your environment:**

* **Angular version:** 2.0.0-rc.4
 
* **Browser:** [all] 
 
* **Language:** [all]

### PR DESCRIPTION
**Pull request by pkozlowski-opensource**
@kara as discussed yesterday trying to help with the forms. Here is my stab at the easy stuff to dive into the code.

Closes #10336

### CODE REVIEW

File: modules/@angular/forms/src/directives/ng_control_status.ts [hunk 1]
```diff
@@ -37,6 +37,9 @@ export class AbstractControlStatus {
   get ngClassInvalid(): boolean {
     return isPresent(this._cd.control) ? this._cd.control.invalid : false;
   }
+  get ngClassPending(): boolean {
+    return isPresent(this._cd.control) ? this._cd.control.pending : false;
```
Comment (vicb) [inline modules/@angular/forms/src/directives/ng_control_status.ts]: remove `isPresent` pls. (yes others too).

Comment (pkozlowski-opensource) [inline modules/@angular/forms/src/directives/ng_control_status.ts]: @vicb done

**Comment by vicb** [comment]
@kara could you please review ?

**Comment by kara** [comment]
LGTM

**Comment by mprobst** [comment]
@IgorMinar can you confirm that this is good to merge?

**Comment by vicb** [comment]
@mprobst milestone 2.0.1 => later

**Comment by IgorMinar** [comment]
we are in feature freeze. we should delay merging this until we branch off a bugfix branch for 2.0/

**Comment by vicb** [comment]
Could you please rebase the PR

**Comment by pkozlowski-opensource** [comment]
@vicb rebased, all back to green.

**Pull request merged**